python/fusion_engine_client/messages/defs.py

```diff
@@ -453,8 +454,13 @@ def find_matching_message_types(cls, pattern: Union[str, List[str]], return_clas
         for pattern in requested_types:
             # Check if pattern is the message integer value.
             try:
-                int_val = int(pattern)
-                result.add(MessageType(int_val))
+                # Allow specifying unknown MessageType values if they are preceded by 'u'.
+                if len(pattern) > 1 and pattern[0] == 'u':
```

Comment:
Is this u prefix really necessary? Any reason not to just treat any manually specified integer as "might be unknown and that's allowable"?

Reply:
The thought was that 99% of the time an external customer would want to filter known messages and this would allow detecting typos.

Reply:
I think in practice this will just be an annoyance more than helpful. The majority of the time someone using this tool is probably going to be specifying message types by name, not number.